v2/tools/generator/internal/codegen/testdata/ArmResource/Arm_test_dependent_resource_and_ownership_crossplane.golden

```diff
@@ -89,40 +89,25 @@ type DList struct {
 }
 
 // +kubebuilder:validation:Enum={"2020-06-01"}
-type ASpecAPIVersion string
+type APIVersion string
 
-const ASpecAPIVersion20200601 = ASpecAPIVersion("2020-06-01")
+const APIVersionValue = APIVersion("2020-06-01")
```

Comment:
I think that this issue ends up fixing #1554

Reply:
I've updated your PR summary to include that (so on merge this should close the issue)